README.md

```diff
@@ -40,10 +40,45 @@ When run locally, auth bypasses Cognito. The frontend mimics login in local stor
 
 ### Running the app locally
 
-- Populate .env files in the root directory and in the `ui-src` directory.
+- Populate .env files in the root directory and in the `ui-src` directory. See Environment Configuration below for up-to-date files.
 - Run all the services locally with the command `./dev local`
 - Troubleshooting: See the Requirements section if the command asks for any prerequisites you don't have installed.
 
+#### Environment Configuration
+
+Root `.env`
+```
+SKIP_PREFLIGHT_CHECK=true
+LOCAL_LOGIN=true
+MEASURE_TABLE_NAME=local-measures
+MEASURE_TABLE_ARN=local_nonsense_if_unset_we_search_CF_for
+measureTableName=local-measures
+DYNAMODB_URL=http://localhost:8000
+API_URL=http://localhost:3030/local
+S3_LOCAL_ENDPOINT=http://localhost:4569
+S3_ATTACHMENTS_BUCKET_NAME=local-uploads
+URL=http://localhost/3000
+DISABLE_ESLINT_PLUGIN=true
+```
+
+/services/ui-src `.env`
+```
+LOCAL_LOGIN=true
+API_REGION=us-east-1
+API_URL=http://localhost:3030
+COGNITO_REGION=us-east-1
+COGNITO_IDENTITY_POOL_ID=us-east-1:61c849d8-f95c-45af-8131-15b75c525034
```

Comment:
All the information here is for a different project than MCR. Can you please correct with the appropriate information for MCR.